jest.config.js

```diff
@@ -69,7 +69,14 @@ module.exports = {
 
   // A set of global variables that need to be available in all test environments
 
-  // globals: {},
+  globals: {
+    'ts-jest': {
+      tsconfig: {
+        allowJs: true,
```

Comment:
This line is for default `react-table` filter functions import to work.

Reply:
If we're allowing JS, I guess there is no need to make setupTests use TS.

Maybe all of this should go in a new PR?

Reply:
Didnt need to allowJs in our tsconfig???

Reply:
For some reason no. I also checked in demo app and everything seems to be working.